arch/platform/srf06-cc26xx/platform.c

```diff
@@ -198,7 +198,13 @@ platform_init_stage_three()
           ti_lib_chipinfo_chip_family_is_cc13xx() == true ? "Yes" : "No",
           ti_lib_chipinfo_supports_ble() == true ? "Yes" : "No",
           ti_lib_chipinfo_supports_proprietary() == true ? "Yes" : "No");
-  LOG_INFO(" RF: Channel %d, PANID 0x%04X\n", chan, pan);
+  LOG_INFO(" RF: Channel %d", chan);
+
+  if(NETSTACK_RADIO.get_value(RADIO_PARAM_PAN_ID, &pan) == RADIO_RESULT_OK) {
+    LOG_INFO(", PANID 0x%04X\n", pan);
+  }
+  LOG_INFO("\n");
```

Comment:
you probably want `LOG_INFO_` here (not to prepend the log prefix)

Reply:
shouldn't this go to `else` to avoid printing `\n` twice?

Reply:
Argh I never meant for the PANID line to have `\n` will fix in a moment

Reply:
That's done now and rebased